recipes/hpv-kite/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Zatrzymaj przy błędzie
+set -o errexit
+
```

Comment:
```suggestion
sed -i 's/id "java"/id "java"\nid "com.github.jk1.dependency-license-report" version "latest.release"/' build.gradle
```
Please add this line so that the `dependency-license-report` plugin can be run.